pkg/restapi/healthcheck/controller.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package healthcheck
```

Comment:
`healthcheck` is very specific. Some names I can think of - ops/performance/metrics. 

May be this is a discussion for a new issue/pr (not a blocker at the moment).